src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -71,7 +72,7 @@ private SwitchBootstraps() {}
     private static final MethodHandle MAPPED_ENUM_LOOKUP;
 
     private static final MethodTypeDesc typesSwitchDescriptor =
```

Comment:
Given that it's a static final, the name should be in uppercase, `TYPES_SWITCH_DESCRIPTOR`